src/main/java/taxi/model/Driver.java

```diff
@@ -6,6 +6,8 @@ public class Driver {
     private Long id;
```

Comment:
lets add `toString()` method to Driver class, so we have appropriate driver description in exceptions messages